lib/routes.js

```diff
@@ -17,7 +17,9 @@ const createReleaseSchema = {
     .additionalProperties(false) // important
     .prop('version', S.string().required())
     .prop('target', S.string())
-    .prop('name', S.string()),
+    .prop('name', S.string())
+    .prop('generateReleaseNotes', S.boolean())
```

Comment:
can't we use the default behavior here in the schema so that we don't have to put it in code?

Reply:
what do you mean by "default behaviour"? Naming the props the same as GitHub expect them (e.g.: generate_release_notes)?

Reply:
default value :)